Update for recent container changes (#21)
vertica/integrators-guide: content/containers/creating-image.md

### Second Stage

> The second stage sets up the OS to be used for the final image. It updates the packages so that we are using the latest ones with all known security vulnerabilities addressed. This stage hands off to the third and final stage to remove any intermediate files needed for the package update.

Reviewer: ```suggestion
The second stage prepares the OS for the final image. It updates packages to address all known security vulnerabilities. This stage hands off to the third and final stage to remove any intermediate files that are needed for the package update.
```

FROM ubuntu:${BASE_OS_VERSION}
```

### Third Stage

The third and final build stage:

- copies only the necessary build artifacts from the first two stages to reduce the number of layers in the final image.
- creates environment variables.
- exposes ports for networking.
- adds metadata to the image.

#### Set Up Stage

> Start with an empty image called scratch and copy everything over from the second stage:

Reviewer: ```suggestion
Start with an empty image called `scratch` and copy everything from the second stage:
```

```
FROM scratch
COPY --from=initial / /
```

> This gets rid of any intermediate layer that was created to update the OS packages.

Reviewer: ```suggestion
This removes any intermediate layers that were created to update the OS packages.
```

  fi \
```

##### Install vim

> Add the following for debugging purposes. The vim package can be quite old. Some security scanners may find vulnerabilities in it. For this reason, we don't include it in the NO_KEYS image, which should be used in environments with strict security requirements.

Reviewer: ```suggestion
Add the Vim text editor for debugging purposes. The `vim` package can be old, and some security scanners might find vulnerabilities in it. For this reason, it is not included it in the NO_KEYS image, which should be used in environments with strict security requirements.
```

### Exposing Ports

> Expose port 5433 for Vertica, 8443 for Vertica's HTTP server and 5444 for vertic agent:

Reviewer: ```suggestion
Expose port 5433 for Vertica, 8443 for Vertica's HTTP server, and 5444 for vertic agent:
```
